### PR TITLE
set base-version in 1.10 workflows

### PR DESCRIPTION
**Pull request by nebril**
0.10.0 cli version is not compatible with 1.10 Cilium and results in
  breakages (e.g. incorrect setting of native routing CIDR in GKE).

### CODE REVIEW

**Review by nbusseneau — APPROVED** [review]
Acking this blindly based on me trusting you did find an issue. Can we open an issue over at `cilium/cilium-cli` with more context?

**Review by tklauser — APPROVED** [review]
Acking this to fix the v1.10 workflows. Ideally we would fix the underlying issue in cilium-cli asap as well and create a new release, so all workflows in cilium/cilium can use the latest cilium-cli version.

File: .github/workflows/conformance-aks-v1.10.yaml [hunk 1]
```diff
@@ -142,7 +142,8 @@ jobs:
             --azure-resource-group ${{ env.name }} \
             --wait=false \
             --rollback=false \
-            --config monitor-aggregation=none"
+            --config monitor-aggregation=none" \
+            --base_version=v1.10.5
```
Comment (tklauser) [inline .github/workflows/conformance-aks-v1.10.yaml]:
```suggestion
            --base-version=v1.10.5
```

same for the other workflows

Comment (nebril) [reply .github/workflows/conformance-aks-v1.10.yaml]: :facepalm: good eye

File: .github/workflows/conformance-aks-v1.10.yaml [hunk 2]
```diff
@@ -142,7 +142,8 @@ jobs:
             --azure-resource-group ${{ env.name }} \
             --wait=false \
             --rollback=false \
-            --config monitor-aggregation=none"
+            --config monitor-aggregation=none" \
+            --base-version=v1.10.5
```
Comment (nbusseneau) [inline .github/workflows/conformance-aks-v1.10.yaml]: @tklauser Would it be better to use `v1.10.0`, or perhaps we can use just `v1.10`? Do we have a scenario where the ~~minor~~ patch version is important? I'm worried we'd need to update this regularly.

**Review by nbusseneau — APPROVED** [review]

**Review by nbusseneau — CHANGES_REQUESTED** [review]
Actually can we update the 1.11 backport workflows as well?

File: .github/workflows/conformance-aks-v1.10.yaml (hunk 2 above)
Comment (nebril) [reply .github/workflows/conformance-aks-v1.10.yaml]: if I set it to v1.10, I get `Creating ConfigMap for Cilium version 1.10.0..` in the logs :/

Comment (tklauser) [reply .github/workflows/conformance-aks-v1.10.yaml]: Yes, let's use either `v1.10` or `v1.10.0` as the base version. `cilium-cli` is only interested in the minor version. I think `v1.10` should work as well from skimming through the code, but I didn't try it out. If `--base-version=v1.10` works, I'd prefer that one over `v1.10.0`.

Comment (tklauser) [reply .github/workflows/conformance-aks-v1.10.yaml]: > if I set it to v1.10, I get `Creating ConfigMap for Cilium version 1.10.0..` in the logs :/

I think that's fine. As mentioned in my comment above, the patch version doesn't matter for cilium-cli.

**Review by pchaigno — APPROVED** [review]

**Review by tklauser — APPROVED** [review]
LGTM, thanks!

**Review by nbusseneau — APPROVED** [review]

**Pull request merged**